packaging/fedora/ksh.spec.in

```diff
@@ -46,7 +46,15 @@ with "sh" (the Bourne Shell).
 %meson_install
 # Allow switching between different ksh implementations (for e.g. mksh) through alternatives.
 mv %{buildroot}/%{_bindir}/ksh %{buildroot}/%{_bindir}/ksh93
-mv %{buildroot}/%{_mandir}/man1/ksh.1.gz %{buildroot}/%{_mandir}/man1/ksh93.1.gz
+# http://mesonbuild.com/Release-notes-for-0-49-0.html#manpages-are-no-longer-compressed-implicitly
+# meson 0.49 does not compress man pages
+if [[ -e  %{buildroot}/%{_mandir}/man1/ksh.1.gz %{buildroot}/%{_mandir}/man1/ksh93.1.gz ]]
+then
+    mv %{buildroot}/%{_mandir}/man1/ksh.1.gz %{buildroot}/%{_mandir}/man1/ksh93.1.gz
```

Comment:
Couldn't this entire `if.. else.. fi` block be simplified to a single `mv` that uses this wildcard statement:

```
mv %{buildroot}/%{_mandir}/man1/ksh.1.* %{buildroot}/%{_mandir}/man1/
```

But since it has been more than 10 years since I dealt with RPM package config files it is likely I have no idea what I'm talking about. In which case this is fine.

Reply:
But we need to rename file from `ksh.1.gz` to `ksh93.1.gz`.